tests/infra/remote.py

```diff
@@ -550,9 +550,14 @@ def __init__(
         exe_files = [self.BIN, lib_path] + self.DEPS
         data_files = [self.ledger_file] if self.ledger_file else []
 
+        # lib_path may be relative or absolute. The remote implementation should
+        # copy (or symlink) to the target workspace, and then node will be able
+        # to reference the destination file locally in the target workspace.
+        enclave_path = os.path.join(".", os.path.basename(lib_path))
+
         cmd = [
             self.BIN,
-            f"--enclave-file={lib_path}",
+            f"--enclave-file={enclave_path}",
             f"--enclave-type={enclave_type}",
```

Comment:
We don't have any tests in the CI that check clients/nodes that run remotely on different machines, so might be worth running a test like to to make sure everything works

Reply:
Quick test suggests it does, thanks!